server/views/pages/bookVisit/selectVisitors.njk

```diff
@@ -48,10 +48,10 @@
             {%- endset %}
 
             {% set visitorHintHtml -%}
-              <p class='banned-hint govuk-!-margin-bottom-0'>
+              <p class='secondary-text govuk-!-margin-bottom-0'>
```

Comment:
Not specifically just this but shouldn't we be using double-quotes in HTML attributes?

Reply:
Fixed!